tests/test_clickhouse.py

```diff
@@ -10,17 +13,20 @@
 @TestScenario
 @Name("test_ch_001. Insert quorum")
 def test_ch_001(self):
-    require_zookeeper()
-
-    create_and_check(
+    util.require_zookeeper()
+    chit_data = manifest.get_chit_data(util.get_full_path("templates/tpl-clickhouse-19.11.yaml"))
+    kubectl.launch(f"delete chit {chit_data['metadata']['name']}", ns=settings.test_namespace)
+    kubectl.create_and_check(
         "configs/test-ch-001-insert-quorum.yaml",
         {
-            "apply_templates": {"templates/tpl-clickhouse-19.11.yaml"},
+            "apply_templates": {"templates/tpl-clickhouse-20.8.yaml"},
```

Comment:
19.11 doesn't support `deduplicate_blocks_in_dependent_materialized_views`, so it worked only because we already have CHIT "clickhouse-version" with much newer clickhouser-server version, which was already applied to clickhouse-operator in previous tests